tools/probes/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
comment on the commit message: can you clarify the usage? It seems to me this is an off-line tool to split the probe data into wave files, but how do we get the probe data in the first place?

Reply:
Yes, offline tool to split the probe data into wave files. This PR is only about this tool, how do we get the probe data depends on kernel implementation (user will use debugfs to add probe points etc.) which probably will be explained in docs.

For this tool the usage is very simple: 
"Probes will extract data for several probe points in one stream
 with extra headers. This app will read the resulting file,
 strip the headers and create wave files for each extracted buffer.

 Usage to parse data and create wave files: ./sof-probes -p data.bin"

What more would you like to add here?